src/main/resources/google_checks.xml

```diff
@@ -105,6 +105,12 @@
       <property name="query" value="//RCURLY[parent::SLIST[count(./*)=1]
                                      or preceding-sibling::*[last()][self::LCURLY]]"/>
     </module>
+    <module name="SuppressionXpathSingleFilter">
+      <property name="checks" value="MethodName"/>
+      <property name="query" value="//METHOD_DEF[
+                                     .//ANNOTATION/IDENT[@text='Test']]//*"/>
+      <property name="message" value="'[a-z][a-z0-9][_a-zA-Z0-9]*'.*"/>
```

Comment:
removed the preceding `.*` from suggested regex:

```
.*'[a-z][a-z0-9][_a-zA-Z0-9]*'.*
```

I don't think it's important